KursaHworks/src/io/codelex/arithmetic/practice/Ex6CozaLozaWoza.java

```diff
@@ -0,0 +1,33 @@
+package io.codelex.arithmetic.practice;
+
+public class Ex6CozaLozaWoza {
+    public static void main(String[] args) {
+        for (int i = 1; i <= 110; i++) {
+            if (i % 33 == 0) {
```

Comment:
Šis IF izskatās "nedaudz" par sarežgītu. :D
Es gribētu intereses pēc zināt, kāpēc tika izmantots 33; 55 un 77 - saprotu, ka tie ir to pamata skaitļu 3; 5; un 7; reizinājumi ar 11.. Piemēram šādi (šo nospēru no Zigmāra):

```
            if (i % 3 == 0) {
                outputStr.append("Coza");
            }
            if (i % 5 == 0) {
                outputStr.append("Loza");
            }
            if (i % 7 == 0) {
                outputStr.append("Woza");
            }
            if (outputStr.toString().equals(" ")) {
                outputStr.append(i);
            }
            if (i % 11 == 0) {
                outputStr.append("\n");
            }
```

Reply:
Jā, kad tā parāda piemēru, tad jāpiekrīt. :D
Kā varbūt pamanīji, es izmantoju println() lai pārnestu jaunā rindā pēc katra vienpadsmitā. Tas tīri tāpēc, ka nebiju vēl to append() metodi pamanījis/izmantojis un šis bija mans risinājums. :D 33/55/77 jo tie ir "unikāli" println() kur katram kkas cits jāizved. Bet nice, tgd būs vēl viens "tool" manā "arsenālā" :D